__tests__/app.test.js

```diff
@@ -102,16 +102,54 @@ describe.only('GET/api/articles:/article_id', () => {
 				);
 			});
 	});
-	test('responds with a 404 status code an an error object when passed an invalid article id', () => {
+	test('responds with a 400 status code an an error object when passed an invalid article id', () => {
 		const ID = 2000;
 		return request(app)
 			.get(`/api/articles/${ID}`)
-			.expect(404)
+			.expect(400)
 			.then(({ body }) => {
 				expect(body).toBeInstanceOf(Object);
 				expect(body).toEqual(
 					expect.objectContaining({
-						msg: 'Route not found',
+						msg: 'Article does not exist',
+					})
+				);
+			});
+	});
+});
+
+describe('GET/api/articles/:article_id/comments', () => {
+	test('responds with a 200 status code and returns an array of ', () => {
+		const ID = 1;
+		return request(app)
+			.get(`/api/articles/${ID}/comments`)
+			.expect(200)
+			.then(({ body }) => {
+				expect(body).toBeInstanceOf(Array);
```

Comment:
Your response should have a label, describing what is being received eg `{comments: [...] }`